src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -474,7 +476,18 @@ export default defineComponent({
       try {
         const { gaAccountId, txHash, nonce } = activeMultisigAccount.value;
         const rawTx = await fetchTransactionByHash(txHash as string);
-        transaction.value = await sendTx(gaAccountId, (rawTx as any).tx, nonce);
+        if (!rawTx) {
+          throw Error('failed to load a transaction');
+        }
+        transaction.value = JSON.parse(
+          JSON.stringify(
+            await sendTx(gaAccountId, rawTx.tx, nonce),
+            // TODO: use bigint in every place for transcation amount and fee
+            (key, value) => typeof value === 'bigint'
+              ? value.toString()
+              : value,
```

Comment:
I'm not sure about this. All `BigInt`, `string` and `number` has the `toString()` method.

Reply:
![image](https://github.com/aeternity/superhero-wallet/assets/9008074/a9bb309c-f618-44aa-a4e0-7d0faff8d8cf)